demo/mock-usb-producer.py

```diff
@@ -24,7 +24,7 @@
 from gi.repository import GLib
 
 from eos_data_distribution.names import SUBSCRIPTIONS_SOMA
-from eos_data_distribution.SimpleStore import Producer as SimpleStoreProducer
+from eos_data_distribution.store import simple_store
```

Comment:
The commit message for this change could do with explaining why this migration is happening.